meta-mbl-distro/classes/mbl-partitions.bbclass

```diff
@@ -219,7 +219,6 @@ MBL_PARTITION_NAMES += "BOOT"
 # ROOT (file system; two banks)
 # ------------------------------------------------------------------------------
 MBL_ROOT_MOUNT_POINT = "/"
-MBL_ROOT_LABEL ?= "rootfs"
```

Comment:
I'm not comfortable with getting rid of the partition label. I do understand that by copying the image file, the label of the partition is get overridden and we need to rely on the partition number store in the `part-info`, instead.

I'm just thinking about user applications for whatever reason will rely on the partition label (or even tests to validate the update). Because we know what partition number is the bank we are flashing to, we could restore the label after the copy.

Reply:
> we could restore the label after the copy.

Not if we start using e.g. squashfs.